addons/mail/static/src/components/welcome_view/welcome_view.js

```diff
@@ -11,21 +11,28 @@ export class WelcomeView extends owl.Component {
      */
     setup() {
         super.setup();
-        useRefToModel({ fieldName: 'guestNameInputRef', modelName: 'mail.welcome_view', propNameAsRecordLocalId: 'localId', refName: 'guestNameInput' });
-        useUpdateToModel({ methodName: 'onComponentUpdate', modelName: 'mail.welcome_view', propNameAsRecordLocalId: 'localId' });
+        useRefToModel({ fieldName: 'guestNameInputRef', modelName: 'mail.welcome_view', propNameAsRecordLocalId: 'welcomeViewLocalId', refName: 'guestNameInput' });
+        useUpdateToModel({ methodName: 'onComponentUpdate', modelName: 'mail.welcome_view', propNameAsRecordLocalId: 'welcomeViewLocalId' });
+    }
+
+    mounted() {
+        if (this.welcomeView.channel.defaultDisplayMode === 'video_full_screen') {
+            this.welcomeView.mediaPreview.enableMicrophone();
+            this.welcomeView.mediaPreview.enableVideo();
+        }
     }
 
     /**
      * @returns {mail.welcome_view}
      */
     get welcomeView() {
-        return this.messaging && this.messaging.models['mail.welcome_view'].get(this.props.localId);
+        return this.messaging && this.messaging.models['mail.welcome_view'].get(this.props.welcomeViewLocalId);
     }
 
 }
 
 Object.assign(WelcomeView, {
-    props: { localId: String },
+    props: { welcomeViewLocalId: String },
```

Comment:
Avoid useless diff, especially if using inconsistent naming (discuss public view is introduced with a props just name "localId").

Reply:
I did it to be consistent with props used in ThreadView.
I'd rather change the name of the prop in DiscussPublicView.